CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Added cost considerations disclaimer in README about Dynatrace Grail data access.
 - Added `dtClientContext` to `execute_dql` tool, to allow usage-monitoring for Grail access.
+- Removed `get_logs_for_entity` tool, in favour of `execute_dql`.
```

Comment:
> favour

That's very British of you ;)

```suggestion
- Removed `get_logs_for_entity` tool, in favor of `execute_dql`.
```

Reply:
My english teacher would be proud of me - they preferred us to use British English whenever possible.